R/navs-legacy.R

```diff
@@ -231,7 +231,9 @@ navbarPage_ <- function(title,
           type = "button",
           class = "navbar-toggle collapsed",
           `data-toggle` = "collapse",
+          `data-bs-toggle` = "collapse",
```

Comment:
Can you add a comment about which versions of Bootstrap each of these attributes is for?